kobo/apps/mfa/models.py

```diff
@@ -9,6 +10,27 @@
 )
 
 
+class KoboMFAPerUserActivation(models.Model):
```

Comment:
i'd suggest `class MfaAllowedForUser`. i don't think we need a `Kobo` prefix in `kobo/apps….py`, at least when there isn't a need to distinguish from some parent class provided by a library. even then, i might suggest `…import MFAMethod as TrenchMFAMethod`.

i like `BlahMfaBlah` more than `BlahMFABlah`. there's precedent for both `CamelXYZCase` and `CamelXyzCase`, where XYZ is some acronym, in our code base; i like the latter more than the former.

Reply:
`MfaAvailableToUser` it will be. 
Maybe `__str__()`  should be left alone and we should start to use `list_field` in the admin model instead.

Reply:
please use `__str__()`, because without it, we see unhelpful things:
```
In [1]: class Thinger:
   ...:     def __init__(self, identifying_attr):
   ...:         self.identifying_attr = identifying_attr
   ...: 

In [2]: t = Thinger("john's thing")

In [3]: print(t)
<__main__.Thinger object at 0x7f86459324f0>
```

Reply:
> I would have preferred to keep the same nomenclature as Trench but it's your call, let's use the latter.

i'm not sure what you mean. `MFA` vs. `Mfa`, or something else?

Reply:
Because Trench is using `MFA` in upper case  (e.g. `MFAMethod`), I would have kept the same. 
But if you prefer the lower case, let's use the lower case ;-)

Reply:
I've started to refactor in this PR. I will create another PR instead.

Reply:
See https://github.com/kobotoolbox/kpi/pull/3918